apps/editor/src/app/routes/imageList.tsx

```diff
@@ -65,13 +134,18 @@ function ImageList() {
   const [currentImage, setCurrentImage] = useState<ImageRefFragment>()
 
   const [activePage, setActivePage] = useState(1)
-  const [limit, setLimit] = useState(5)
+  const [limit, setLimit] = useState(DEFAULT_TABLE_IMAGE_PAGE_SIZES[0])
 
   const [isUploadModalOpen, setUploadModalOpen] = useState(isUploadRoute)
   const [isEditModalOpen, setEditModalOpen] = useState(isEditRoute)
 
   const [editID, setEditID] = useState<string | undefined>(isEditRoute ? id : undefined)
 
+  const isGridKey = 'isGrid'
+  const [isGrid, setIsGrid] = useState<boolean>(
+    JSON.parse(localStorage.getItem(isGridKey) || 'true')
```

Comment:
I'd maybe think about moving the value of isGridKey to utility.ts line 10 (enum LocalStorageKey).